README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Node CI](https://github.com/nodejs/undici/actions/workflows/nodejs.yml/badge.svg)](https://github.com/nodejs/undici/actions/workflows/nodejs.yml) [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/) [![npm version](https://badge.fury.io/js/undici.svg)](https://badge.fury.io/js/undici) [![codecov](https://codecov.io/gh/nodejs/undici/branch/main/graph/badge.svg?token=yZL6LtXkOA)](https://codecov.io/gh/nodejs/undici)
 
-A HTTP/1.1 client, written from scratch for Node.js.
+An HTTP/1.1 client, written from scratch for Node.js.
```

Comment:
This is incorrect

Reply:
I guess it depends on how you pronounce "HTTP", if it's "ach-tee-tee-pee" then it requires the "an".
It's also already used in `package.json`:

https://github.com/nodejs/undici/blob/f70ac7f57ae1b95b8140bcbe408274961fc1ac92/package.json#L4